src/lib/markdown.ts

```diff
@@ -125,6 +125,17 @@ export const renderer = {
 
     return `<h${level} id="${escapedText}">${text}</h${level}>`;
   },
+  listitem(text: string) {
+    const hasLineBreaks = text.trim().indexOf("\n");
+    if (hasLineBreaks === -1) {
+      return "<li>" + text + "</li>";
+    }
+    const [first, ...next] = text.trim().split("\n");
```

Comment:
This should probably be `remaining` instead of `next`.

Reply:
Ok did fix this and also added some more template literals